resources/dicts/patrols/plains/border/greenleaf.json

```diff
@@ -31,13 +36,4753 @@
+                "exp": 15,
+                "weight": 5,
+                "art": "gen_happy_cat3",
+                "new_cat": [
+                    ["age:adult", "loner", "meeting"]
+                ],
+                "relationships": [
+                    {
+                        "cats_from": ["n_c:0"],
+                        "cats_to": ["p_l"],
+                        "mutual": false,
+                        "values": ["platonic", "trust", "comfort"],
+                        "amount": 20
+                    },
+                    {
+                        "cats_from": ["p_l"],
+                        "cats_to": ["n_c:0"],
+                        "mutual": false,
+                        "values": ["platonic", "dislike"],
+                        "amount": 10
+                    }
+                ],
+                "outsider_rep": 1
+            },
+            {
+                "text": "s_c stares back, unimpressed. Does n_c:0 really think s_c can't see, or is going to ignore, the piece of prey half concealed behind {PRONOUN/n_c:0/object}? Well. Okay, you can't blame n_c:0 for trying, haha! s_c escorts the minx to the border, peacefully but <i>firmly</i> confistcating the prey {PRONOUN/n_c:0/subject} poached. Shoo.",
+                "exp": 15,
+                "weight": 20,
+                "art": "gen_train_talkingWL",
+                "can_have_stat": ["p_l"],
+                "stat_trait": [
+                    "careful",
+                    "calm",
+                    "cold",
+                    "wise"
+                ],
+                "new_cat": [
+                    ["age:adult", "loner", "meeting"]
+                ],
+                "prey": ["medium"],
+                "relationships": [
+                    {
+                        "cats_from": ["n_c:0"],
+                        "cats_to": ["p_l"],
+                        "mutual": false,
+                        "values": ["platonic", "trust", "romantic"],
+                        "amount": 20
+                    },
+                    {
+                        "cats_from": ["p_l"],
+                        "cats_to": ["n_c:0"],
+                        "mutual": false,
+                        "values": ["platonic", "dislike", "romantic"],
+                        "amount": 10
+                    }
+                ],
+                "outsider_rep": 1
+            },
+            {
+                "text": "Wow, if s_c was a fly, {PRONOUN/s_c/subject}'d be all over n_c:0, because {PRONOUN/n_c:0/subject}{VERB/n_c:0/'re/'s} hot dung, s_c snarks back. Now, give that prey back, before anyone has to draw their claws. Ouch, n_c:0 laughs. Beautiful <i>and</i> clever, s_c is the whole package. Pity about the Clancat bit. The banter continues as s_c escorts n_c:0 to the broder, lingering just a little too long.",
```

Comment:
```suggestion
                "text": "Wow, if s_c was a fly, {PRONOUN/s_c/subject}'d be all over n_c:0, because {PRONOUN/n_c:0/subject}{VERB/n_c:0/'re/'s} hot dung, s_c snarks back. Now, give that prey back, before anyone has to draw their claws. Ouch, n_c:0 laughs. Beautiful <i>and</i> clever, s_c is the whole package. Pity about the Clancat bit. The banter continues as s_c escorts n_c:0 to the border, lingering just a little too long.",
```